manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -32,6 +32,8 @@ pilot:
 
   env: {}
 
+  affinity: {}
```

Comment:
I think tolerations were added alongside affinity, and neither was updated in the IstioOperator file. Should we add that as well?

https://github.com/istio/istio/blob/9852844f311e27b55ffaa0ac71b8968b44b453eb/manifests/charts/istio-control/istio-discovery/templates/deployment.yaml#L69-L71

Reply:
Should I add `tolerations: []` in [manifests/charts/istio-control/istio-discovery/values.yaml](https://github.com/istio/istio/pull/47639/files/a3399f3d2159f4baae63dd342db267d8becc00c8#diff-72aceeff43207b39e2ff9d8d20cb5e26eafe3c7a07d3e648099887b3c47f549e) File ?

Reply:
I'd open another PR for that.